frontend/src/pages/index.js

```diff
@@ -2,12 +2,19 @@ import { useKeycloak } from '@react-keycloak/web';
 import React from 'react';
 import EligibilityPageContainer from '../components/Eligibility/EligibilityPageContainer';
 import Layout from '../components/Layout';
+import {Helmet} from "react-helmet";
 
 function Index() {
   const { keycloak } = useKeycloak();
 
   return (
     <div className="app">
+       <Helmet>
+            <title>Passenger vehicle rebate log in – CleanBC Go Electric</title>
```

Comment:
This will just overwrite the title 4 times on the index page. Each specific page component needs to be updated with only its title.

- Description Pages
- Form Pages
- BCeID explanation page

You'll have to navigate around the app and check the titles are updating as the content changes.

Reply:
@naomiaro I am unable to navigate to other pages defined in the router. All the mentioned URL's redirect again to the Home page. Kindly let me know how to navigate and which title to be assigned to respective page.